document/core/appendix/properties.rst

```diff
@@ -157,8 +157,6 @@ Module instances are classified by *module contexts*, which are regular :ref:`co
 :ref:`Host Function Instances <syntax-funcinst>` :math:`\{\FITYPE~\functype, \FIHOSTCODE~\X{hf}\}`
 ..................................................................................................
 
-* The :ref:`function type <syntax-functype>` :math:`\functype` must be :ref:`valid <valid-functype>`.
```

Comment:
Hm, not sure why this is removed on the multi-value branch, perhaps some earlier merge mistake. I created a [PR on the repo](https://github.com/WebAssembly/multi-value/pull/43) to fix it.